test/jdk/com/sun/jdi/EATests.java

```diff
@@ -232,14 +200,11 @@ public static void main(String[] args) {
 
         // Relocking test cases
         new EARelockingSimpleTarget()                                                       .run();
-        new EARelockingSimple_2Target()                                                     .run();
```

Comment:
I know all the tests that were removed mention biased locking in the comments, but do they require biased locking to function properly? I'm just wondering if we might get better EA test coverage if they are left in place.

Reply:
They are trying to exercise some biased locking specific paths, but maybe @reinrich can comment on wether it is worth keeping them since he wrote the tests.

Reply:
The test cases are very specific to biased locking. It is not worth keeping them.